app/auth/keycloak_auth.py

```diff
@@ -25,8 +24,4 @@ class KeycloakAccessToken():
 
     def process(self, request, headers):
 
```

Comment:
So now this, does nothing?

Reply:
As keycloak tokens are already present in the headers (used by the UI too), this one has nothing to do, yes.

Reply:
But you think we can find a use for this function? Else we might just refactor it out.

Reply:
There is no superclass implementing the defaults, so the method has to exist here. We will see if it makes sense to have a base_auth class in the future.